Sources/Core/Schema.swift

```diff
@@ -90,12 +90,9 @@ func decodeRef(from source: URL, with ref: String) -> URL {
         // Local URL
         return URL(string:ref, relativeTo:source)!
     } else {
-        var baseUrl = source.deletingLastPathComponent()
-        if baseUrl.path == "." {
-            baseUrl = URL(fileURLWithPath: (baseUrl.path))
-        }
+        let baseUrl = source.deletingLastPathComponent()
         let lastPathComponentString = URL(string: ref)?.pathComponents.last
-        return URL(string:lastPathComponentString!, relativeTo:baseUrl)!
+        return URL(string:baseUrl.path + "/" + lastPathComponentString!)!
```

Comment:
Maybe better here to use
```Swift
return URL(string:baseUrl.path).URLByAppendingPathComponent(lastPathComponentString!)
// OR if baseUrl is an URL
return baseUrl.URLByAppendingPathComponent(lastPathComponentString!)
```

Connecting file paths via`/` is not safe cross platforms.